content-security-policy/support/testharness-helper.sub.js

```diff
@@ -5,8 +5,30 @@ const Host = {
 
 const PolicyHeader = {
   CSP: "echo-policy.py?policy=",
+  CSP_MULTIPLE: "echo-policy-multiple.py",
   EMBEDDING_CSP: "echo-embedding-csp.py",
-} 
+  ALLOW_CSP_FROM: "echo-allow-csp-from.py",
+};
+
+const IframeLoad = {
+  EXPECT_BLOCK: true,
+  EXPECT_LOAD: false,
+};
+
+function getOrigin() {
+  var url = new URL("http://{{host}}:{{ports[http][0]}}/");
+  return url.toString();
+}
+
+function getCrossOrigin() {
+  var url = new URL("http://{{domains[天気の良い日]}}:{{ports[http][0]}}/");
+  return url.toString();
+}
+
+function getSecureCrossOrigin() {
+  var url = new URL("https://{{domains[天気の良い日]}}:{{ports[http][0]}}/");
```

Comment:
Can you add a comment explaining this? I never would have noticed it if you didn't point it out (and I'm still not sure I understand why it's necessary).